test/framework/kogitoruntime.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
+)
+
+// DeployRuntimeService deploy a Kogito service
+func DeployRuntimeService(namespace string, installerType InstallerType, kogitoRuntime *v1alpha1.KogitoRuntime) error {
```

Comment:
Implementations in this file can be partly replaced by functions from https://github.com/kiegroup/kogito-cloud-operator/blob/master/test/framework/kogitoserviceutils.go , though it can be addressed lately.

Reply:
Yes, maybe. So far, the kogito service utils was only used by data-index, management console and jobs service, not for kogito app. But it's true that can be used for kogito app and kogito runtime too. 
Agreed that can be done once we remove the kogitoapp support and with some extra refactor.